src/pages/workspace/qbo/PolicyQuickbooksImportPage.tsx

```diff
@@ -0,0 +1,126 @@
+import React, {useCallback} from 'react';
+import HeaderWithBackButton from '@components/HeaderWithBackButton';
+import MenuItemWithTopDescription from '@components/MenuItemWithTopDescription';
+import OfflineWithFeedback from '@components/OfflineWithFeedback';
+import ScreenWrapper from '@components/ScreenWrapper';
+import ScrollView from '@components/ScrollView';
+import Text from '@components/Text';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@hooks/useThemeStyles';
+import Navigation from '@libs/Navigation/Navigation';
+import withPolicy from '@pages/workspace/withPolicy';
+import type {WithPolicyProps} from '@pages/workspace/withPolicy';
+import CONST from '@src/CONST';
+import ROUTES from '@src/ROUTES';
+
+const NAVIGATION_KEYS = {
+    ACCOUNTS: 'accounts',
+    CLASSES: 'classes',
+    CUSTOMERS: 'customers',
+    LOCATIONS: 'locations',
+    TAXES: 'taxes',
+};
+
+function PolicyQuickbooksImportPage({policy}: WithPolicyProps) {
+    const {translate} = useLocalize();
+    const styles = useThemeStyles();
+    const quickbooksOnlineConfigTitles = {
+        DEFAULT: translate('workspace.qbo.imported'),
+        true: translate('workspace.qbo.imported'),
+        false: translate('workspace.qbo.notImported'),
+        NONE: translate('workspace.qbo.notImported'),
+        TAG: translate('workspace.qbo.importedAsTags'),
+        REPORT_FIELD: translate('workspace.qbo.importedAsReportFields'),
+    };
+    const {syncClasses, syncCustomers, syncLocations, syncTaxes, syncAccounts} = policy?.connections?.quickbooksOnline?.config ?? {};
+
+    const onPressConfigOption = useCallback(
+        (option: string) => {
+            const policyID = policy?.id ?? '';
+            switch (option) {
+                case NAVIGATION_KEYS.ACCOUNTS:
+                    Navigation.navigate(ROUTES.POLICY_ACCOUNTING_QUICKBOOKSONLINE_CHART_OF_ACCOUNTS.getRoute(policyID));
+                    break;
+                case NAVIGATION_KEYS.CLASSES:
+                    Navigation.navigate(ROUTES.POLICY_ACCOUNTING_QUICKBOOKSONLINE_CLASSES.getRoute(policyID));
+                    break;
+                case NAVIGATION_KEYS.CUSTOMERS:
+                    Navigation.navigate(ROUTES.POLICY_ACCOUNTING_QUICKBOOKSONLINE_CUSTOMER.getRoute(policyID));
+                    break;
+                case NAVIGATION_KEYS.LOCATIONS:
+                    Navigation.navigate(ROUTES.POLICY_ACCOUNTING_QUICKBOOKSONLINE_LOCATIONS.getRoute(policyID));
+                    break;
+                case NAVIGATION_KEYS.TAXES:
+                    Navigation.navigate(ROUTES.POLICY_ACCOUNTING_QUICKBOOKSONLINE_TAXES.getRoute(policyID));
+                    break;
+                default:
+                    break;
+            }
+        },
+        [policy?.id],
+    );
+
+    const sections = [
+        {
+            description: translate('workspace.qbo.accounts'),
+            navigationKey: NAVIGATION_KEYS.ACCOUNTS,
```

Comment:
We could simplify this a bit by using the same approach we're using for `MenuItems`
```suggestion
            action: () => Navigation.navigate(ROUTES.POLICY_ACCOUNTING_QUICKBOOKSONLINE_CHART_OF_ACCOUNTS.getRoute(policyID)),
```